### PR TITLE
Statsify.StatsifyOverlay version 2.7.2

### PR DESCRIPTION
**Pull request by TakenMC**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17579)

### CODE REVIEW

File: manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Statsify.StatsifyOverlay
+PackageVersion: 2.7.2
+PackageLocale: en-US
+Publisher: Statsify
+PackageName: Statsify Overlay
```
Comment (OfficialEsco) [inline manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.locale.en-US.yaml:7]:
```suggestion
Publisher: Statsify Inc
PublisherUrl: https://statsify.net
PackageName: satisify-overlay
PackageUrl: https://statsify.net/overlay
Moniker: satisify overlay
```

File: manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Statsify.StatsifyOverlay
+PackageVersion: 2.7.2
+PackageLocale: en-US
+Publisher: Statsify
+PackageName: Statsify Overlay
+License: GNU License
+ShortDescription: Statsify Overlay
```
Comment (OfficialEsco) [inline manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Statsify Overlay for Hypixel Bedwars
Description: 
```
Can you add a better ShortDescription and Description?

File: manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: Statsify.StatsifyOverlay
+PackageVersion: 2.7.2
+InstallerType: "msix"
```
Comment (OfficialEsco) [inline manifests/s/Statsify/StatsifyOverlay/2.7.2/Statsify.StatsifyOverlay.installer.yaml:5]:
```suggestion
Platform:
- Windows.Desktop
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
👀

**Comment by TakenMC** [comment]
Requests approved
Anything else

**Comment by OfficialEsco** [comment]
> Requests approved
> Anything else

Naaaaah you good, i went a bit overboard, the bot still have to rerun the test tho, so unless i did a typo it should be fine in 30min ish and ill approve it

**Comment by OfficialEsco** [comment]
Hold up, you COULD make a better ShortDescription and Description as i requested since am not 100% sure what the overlay is for/does

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**